scripts/publish-image-runtime

```diff
@@ -5,4 +5,5 @@ cd $(dirname $0)/..
 
 source ./scripts/version.sh
 
-docker image push ${REPO}/${PROG}-runtime:${DOCKERIZED_VERSION}-${GOARCH}
+docker image push ${REPO}/${PROG}-runtime:${DOCKERIZED_VERSION}-${GOOS}-${GOARCH}
+docker image push ${REPO}/${PROG}-runtime:${DOCKERIZED_VERSION}-windows-amd64
```

Comment:
We're not supporting Windows for 1.20, are we?

Reply:
we are not, im still getting to this one